app/src/main/java/com/architectcoders/aacboard/ui/fragments/viewmodel/SearchPictogramsViewModel.kt

```diff
@@ -0,0 +1,70 @@
+package com.architectcoders.aacboard.ui.fragments.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.architectcoders.aacboard.data.PictogramUI
+import com.architectcoders.aacboard.data.toUIPictogram
+import com.architectcoders.aacboard.domain.data.Error
+import com.architectcoders.aacboard.domain.data.Response.Failure
+import com.architectcoders.aacboard.domain.data.Response.Success
+import com.architectcoders.aacboard.domain.use_case.location.GetLastUserRegionUseCase
+import com.architectcoders.aacboard.domain.use_case.search.SearchPictogramsUseCase
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+class SearchPictogramsViewModel(
+    val searchPictogramsUseCase: SearchPictogramsUseCase,
+    val getLastUserRegionUseCase: GetLastUserRegionUseCase
```

Comment:
Both use case should be private to avoid exposure to outsiders.

Reply:
ok